src/main.rs

```diff
@@ -50,6 +51,9 @@ async fn log_task_completion(name: &str, task: JoinHandle<Result<(), Error>>) {
 async fn main() -> Result<(), Error> {
     let args = Args::parse();
     let config = Arc::new(Config::from_args(&args)?);
+    let runtime_config = Arc::new(RuntimeConfigManager::new(
+        config.runtime_config_path.clone(),
+    ));
```

Comment:
Will this get pushed into the `InflightActivationStore`?

Reply:
If we eventually require the it to need runtime configs, then yes. Otherwise, dropping tasks via killswitching happens outside of `InflightActivationStore`